### ISSUE TITLE
Create validator for unnecessary meta tags

### ISSUE THREAD

**Issue by capjamesg** [indieweb/indiewebify-me#104]
@tantek and I, among other participants in the IndieWeb community, were discussing unnecessary meta tags on websites. Unnecessary meta tags may be duplicative, oriented around a particular service, or provide little to no use to users.

It would be great if the indiewebify tool had a feature to audit the meta tags on a website and make suggestions about what could be removed that may not be useful. For instance, I decided to remove Twitter image and description tags because I had the same information in other meta tags.

The tool should make suggestions rather than warnings. The goal would be to encourage people to consider the value of the meta tags they have to them and their audience.

**Comment by jamietanna** [comment]
Can we make sure there are warnings about when ie removing the Twitter image metadata impacts functionality, such a sharing to Twitter, Slack and other platforms? Often the tags have a reason, and although it's a good thing to strive for limiting duplication, it's worth highlighting its a tradeoff

**Comment by gRegorLove** [comment]
Yeah, I would be OK with this as long as the advice carefully follows what we've [documented on OGP](https://indieweb.org/The-Open-Graph-protocol#How_To) and makes it clear the trade-offs. Some of these are more like "You _can_ do this _if_..." rather than "You _should_ remove...". See also [this post](https://meiert.com/en/blog/minimal-social-markup/) that was recently added to https://indieweb.org/link-preview#See_Also
